coral-hive/src/main/antlr/roots/com/linkedin/coral/hive/hive2rel/parsetree/parser/HiveParser.g

```diff
@@ -653,7 +653,11 @@ import org.apache.hadoop.hive.conf.HiveConf;
     this.hiveConf = hiveConf;
   }
   protected boolean useSQL11ReservedKeywordsForIdentifier() {
-    return !HiveConf.getBoolVar(hiveConf, HiveConf.ConfVars.HIVE_SUPPORT_SQL11_RESERVED_KEYWORDS);
+    try {
+      return !HiveConf.getBoolVar(hiveConf, HiveConf.ConfVars.HIVE_SUPPORT_SQL11_RESERVED_KEYWORDS);
+    } catch (Throwable throwable) {
+      return false;
```

Comment:
can we please LOG a warn with the exception here?

Reply:
Thank you for suggestion, I will open another PR to add the log and run integration test for it.